docs/tutorials/python/linear-regression.md

```diff
@@ -30,60 +28,57 @@ eval_label = np.array([11,26,16])
 ```
 
 Once we have the data ready, we need to put it into an iterator and specify
-parameters such as the 'batch_size', and 'shuffle' which will determine the size
-of data the iterator feeds during each pass, and whether or not the data will be
-shuffled respectively.
+parameters such as `batch_size` and `shuffle`. `batch_size` determines the amount
+of data the iterator feeds during each time and `shuffle` tells whether or not to shuffle  input data.
```

Comment:
"the amount of data" and "during each time" are both a bit vague. 
Please change to "the number of examples shown to the model each time we update its parameters"

Also "`shuffle` tells whether or not to shuffle" is a bit circular. Can you be more specific - does it say whetehr the data will be shuffled *ever* or whether or not it will be shuffled before each update?